networking/nw-create-custom-ingress-controller

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-create-custom-ingress-controller_{context}"]
+= Creating a custom Ingress Controller
+
+When creating an custom Ingress Controller, 
+
+.Prerequisites
+
+* Must have cluster-admin permissions to the cluster
+* Create a custom wildcard certificate for the custom domain.
+?Do we have a link to this elsewhere...(# oc create secret tls custom-ingress-custom-certs -n openshift-ingress --cert=<certificate.crt> --key=<key.key>)
+
+.Procedure
+
+. Define the ingress controller in a file as template:
++
+[source,yaml]
+----
+# cat custom-ingress-controller.yaml
+
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+name: custom
+namespace: openshift-ingress-operator
+spec:
+defaultCertificate:
+    name: custom-ingress-custom-certs
+domain: "custom-domain.mycompany.com"
+nodePlacement:
+    nodeSelector:
+    matchLabels:
+        ingress-custom-controller: "true"
```

Comment:
These four lines (from `nodePlacement` through `ingress-custom-controller: "true"`) can be omitted as this is intended to be a minimal example.

Reply:
ty